prow/plugins.yaml

```diff
@@ -126,12 +126,20 @@ slack:
     whitelist:
     - k8s-ci-robot
 
-milestone:
-  # You can curl the following endpoint in order to determine the github ID of your team
-  # responsible for maintaining the milestones:
-  # curl -H "Authorization: token <token>" https://api.github.com/orgs/<org-name>/teams
-  maintainers_id: 2460384
-  maintainers_team: kubernetes-milestone-maintainers
+repo_milestone:
+  # Default maintainer
+  '':
+    milestone:
+      # You can curl the following endpoint in order to determine the github ID of your team
+      # responsible for maintaining the milestones:
+      # curl -H "Authorization: token <token>" https://api.github.com/orgs/<org-name>/teams
+      maintainers_id: 2460384
+      maintainers_team: kubernetes-milestone-maintainers
+
+  kubernetes/test-infra:
+    milestone:
+      maintainers_id: 2460384
+      maintainers_team: kubernetes-milestone-maintainers
```

Comment:
if this were for the `kubernetes-sigs/kubebuilder` repo, would it be looking for `@kubernetes-sigs/kubernetes-milestone-maintainers` or `@kubernetes/kubernetes-milestone-maintainers`?

Reply:
IIUC, it's uniquely determined only by using the ID.

Reply:
So this would then be only cosmetic (the links and comments), rather than the actual API call?